static/js/util/profile_edit.js

```diff
@@ -220,6 +221,57 @@ export function boundCheckbox(keySet: string[], label: string|React$Element<*>):
   );
 }
 
+const onTelChange = R.curry((
+  keySet,
+  countryKeySet,
+  profile,
+  updateProfile,
+  validator,
+  newPhoneNumber,
+  countryObj
+) => {
+  let currentPhoneNumber = _.get(profile, keySet);
+  if ( currentPhoneNumber !== newPhoneNumber ) {
+    let clone = _.cloneDeep(profile);
+    _.set(clone, keySet, newPhoneNumber);
+    _.set(clone, countryKeySet, countryObj.iso2);
+    updateProfile(clone, validator);
+  }
+});
+
+export function boundTelephoneInput(keySet: string[], countryKeySet: string[]): React$Element<*> {
+  const {
+    profile,
+    errors,
+    updateProfile,
+    validator,
+    updateProfileValidation,
+    updateValidationVisibility,
+  } = this.props;
+
+  let onBlur = () => {
+    updateValidationVisibility(keySet);
+    updateProfileValidation(profile, validator);
+    sendFormFieldEvent(keySet);
+  };
+
+  let currentCountry = R.toLower(R.defaultTo("", _.get(profile,  ['country'])));
```

Comment:
This is probably worth a discussion outside this PR, but code that uses both Ramda and lodash feels weird to me. Long-term we might want to figure out if we try to standardize around usage of one. This could probably be `R.toLower(R.pathOr("", ['country'], profile))` if you want to go pure-Ramda.